README.md

```diff
@@ -56,6 +56,11 @@ Then on server:
 flask db upgrade
 ```
 
+Override `ALLOWED_REGISTRATION_DOMAINS` const in `config.py` to allow some group of users to access your app:
+```
+ALLOWED_REGISTRATION_DOMAINS = 'gmail.com'
```

Comment:
The only note in this PR to fix: you named this as domainS. This means that you should provide multiple domains. So, this parameter should be a list of values.